yesod-paginator.cabal

```diff
@@ -1,11 +1,11 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
+-- This file has been generated from package.yaml by hpack version 0.34.5.
```

Comment:
Not sure why this changed.

Reply:
It doesn't matter, this churns all the time. It's a function of `stack` version, I think.